x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -165,6 +165,7 @@ export function App({
     setIndicateNoData(true);
   }, [setIndicateNoData]);
 
+  // TODO: Remove this comment after testing
```

Comment:
Not sure it is strictly related to this PR, but from my quick testing I see these page names from different flows:
* `Create visualization` in dashboard, opens a Lens editor => `application:lens:editor` ✅ 
* `Create ES|QL` panel in dashboard => `application:dashboards:app`
* `Edit inline` in a Dashboard => `application:dashboard:app` ❓ 
* `Edit in editor` from a Dashboard inline editing panel => `application:lens:editor` ✅ 
* `Edit visualization` in Discover => `application:lens:/edit_by_value` ❓